terraform/a2ha-terraform/modules/s3/inputs.tf

```diff
@@ -1,3 +1,8 @@
+variable "aws_region" {
+  default     = ""
+  description = "The name of the selected AWS region / datacenter."
+}
+
```

Comment:
is the terrafrom is not using aws_s3_bucketName then  we can remove it

Reply:
aws_s3_bucketName is used in terraform 'null_resource.createS3bucket' resource